scripts/gradle/gradle-check.sh

```diff
@@ -20,6 +20,39 @@ TRIGGER_TOKEN=$1
 PR_TITLE_NEW=`echo $pr_title | tr -dc '[:alnum:] ' | tr '[:upper:]' '[:lower:]'`
 PAYLOAD_JSON="{\"pr_from_sha\": \"$pr_from_sha\", \"pr_from_clone_url\": \"$pr_from_clone_url\", \"pr_to_clone_url\": \"$pr_to_clone_url\", \"pr_title\": \"$PR_TITLE_NEW\", \"pr_number\": \"$pr_number\"}"
 
+perform_curl_and_process_with_jq() {
+    local url=$1
+    local jq_filter=$2
+    local max_retries=$3
+    local count=0
+    local success=false
+
+    while [ $count -lt $max_retries ]; do
```

Comment:
Within the `[]` block, use double quotes to wrap all the variables.

Reply:
thanks for noticing that ,, let me do double quotes here